apps/api/test/04-report-4.e2e-spec.ts

```diff
@@ -0,0 +1,200 @@
+
+    const licenseBefore = reportFourNumberOfApplicants(
+      before,
+      STATUS.BCCNM_FULL_LICENCE_LPN,
+      false,
+    );
+    const { id } = await addJob(applicantIdsOldProcess[1], {
+      ha_pcn: haPcn,
+      job_id: '46897',
+      recruiter_name: 'Tester',
+    });
+    await addMilestone(applicantIdsOldProcess[1], id, await getStatus(STATUS.JOB_OFFER_ACCEPTED));
+
+    const { body: after } = await request(app.getHttpServer()).get(URLS.REPORT4);
+
+    const licenseAfter = reportFourNumberOfApplicants(after, STATUS.BCCNM_FULL_LICENCE_LPN, false);
+    // When the applicant gets hired they should be subtracted from the report value
+    expect(licenseAfter).toBe((parseInt(licenseBefore!) - 1).toString());
+  });
+
+  it('Applicant with all 4 licenses and check granted licensure', async () => {
+    const { body: before } = await request(app.getHttpServer()).get(URLS.REPORT4);
+
+    for (const stat of BCCNM_LICENSE) {
+      const licenseBefore = reportFourNumberOfApplicants(before, stat, false);
+      const milestone = await addMilestone(applicantIdsOldProcess[2], '', await getStatus(stat));
+      const { body: afterLicense } = await request(app.getHttpServer()).get(URLS.REPORT4);
+      const licenseAfter = reportFourNumberOfApplicants(afterLicense, stat, false);
+      expect(licenseAfter).toBe((parseInt(licenseBefore!) + 1).toString());
+      await deleteApplicantStatus(applicantIdsOldProcess[2], milestone.id);
+    }
+  });
+
+  it('NCAS value increases on report for each status Completed CBA or Completed SLA', async () => {
+    const { body: before } = await request(app.getHttpServer()).get(URLS.REPORT4);
+    const ncasBefore = reportFourNumberOfApplicants(before, STATUS.REFERRED_TO_NCAS, false);
+
+    await addMilestone(applicantIdsOldProcess[3], '', await getStatus(STATUS.COMPLETED_CBA));
+    await addMilestone(applicantIdsOldProcess[4], '', await getStatus(STATUS.COMPLETED_SLA));
+
+    const { body: after } = await request(app.getHttpServer()).get(URLS.REPORT4);
+    const ncasAfter = reportFourNumberOfApplicants(after, STATUS.REFERRED_TO_NCAS, false);
+
+    expect(ncasBefore).toStrictEqual((parseInt(ncasAfter!) - 2).toString());
+  });
+
+  it('NNAS report value increases for each status Received NNAS Report or Submitted documents', async () => {
+    const { body: before } = await request(app.getHttpServer()).get(URLS.REPORT4);
+    const nnasBefore = reportFourNumberOfApplicants(before, STATUS.APPLIED_TO_NNAS, false);
+
+    await addMilestone(applicantIdsOldProcess[5], '', await getStatus(STATUS.RECEIVED_NNAS_REPORT));
+    await addMilestone(applicantIdsOldProcess[6], '', await getStatus(STATUS.SUBMITTED_DOCUMENTS));
+
+    const { body: after } = await request(app.getHttpServer()).get(URLS.REPORT4);
+    const nnasAfter = reportFourNumberOfApplicants(after, STATUS.APPLIED_TO_NNAS, false);
+
+    expect(nnasBefore).toStrictEqual((parseInt(nnasAfter!) - 2).toString());
```

Comment:
I prefer `Number` rather than parseInt, it's more strict when converting strings to numbers.  Just in the case we get back a weird string

Reply:
I vote for `+nnasAfter`

Reply:
or that, I think we use `+` majority of the time anyways, so best to stick with what we're already using (need to update a few reports, been using Number :P)